pages/item/_itemId/index.vue

```diff
@@ -13,7 +13,7 @@
             {{ item.Name }}
           </h1>
           <h2
-            v-if="item.OriginalTitle"
+            v-if="item.OriginalTitle !== item.Name"
```

Comment:
This changes the way the condition works.

Example. Before:
```
OriginalTitle = undefined => not shown
OriginalTitle = null => not shown
OriginalTitle = "Lorem Ipsum" => Shown
```

Now:
```
OriginalTitle = null && Name = "Lorem Ipsum" => Shown
OriginalTitle = undefined && Name = "Lorem Ipsum" => Shown
```

The condition should actually be:
```
item.OriginalTitle && item.OriginalTitle !== item.Name
```

Reply:
Thanks, I was thinking about that myself, but forgot later haha :)